### PR TITLE
Translate hard-coded 'Enable labeling' string

### LINKED ISSUE #3813: Create a workaround for "Enable labelling" to be translatable
Related: https://github.com/trezor/trezor-firmware/issues/3697

### PR DESCRIPTION
**Pull request by bieleluk**
Fixes #3813

### CODE REVIEW

File: core/src/apps/misc/cipher_key_value.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ async def cipher_key_value(msg: CipherKeyValue) -> CipheredKeyValue:
         await confirm_action(
             "cipher_key_value",
             title,
-            description=msg.key,
+            description=TR.misc__enable_labeling,
```
Comment (prusnak) [inline core/src/apps/misc/cipher_key_value.py:42]: This will not work as intended when we are not using cipher_key_value to enable labeling

**Review by prusnak — CHANGES_REQUESTED** [review]

File: core/src/apps/misc/cipher_key_value.py (hunk 1 above)
Comment (matejcik) [reply core/src/apps/misc/cipher_key_value.py]: fixed in force-push

**Comment by matejcik** [comment]
please add a changelog entry

File: core/src/apps/misc/cipher_key_value.py [hunk 2]
```diff
@@ -29,17 +29,19 @@ async def cipher_key_value(msg: CipherKeyValue) -> CipheredKeyValue:
         if msg.key == "Enable labeling?":
             title = TR.misc__title_suite_labeling
             verb = TR.buttons__enable
+            description = TR.misc__enable_labeling
```
Comment (prusnak) [inline core/src/apps/misc/cipher_key_value.py:32]:
```suggestion
            description = TR.misc__enable_labeling
            verb = TR.buttons__enable
```

File: core/src/apps/misc/cipher_key_value.py [hunk 3]
```diff
@@ -29,17 +29,19 @@ async def cipher_key_value(msg: CipherKeyValue) -> CipheredKeyValue:
...
         else:
             if encrypt:
                 title = TR.misc__encrypt_value
             else:
                 title = TR.misc__decrypt_value
             verb = TR.buttons__confirm
+            description = msg.key
```
Comment (prusnak) [inline core/src/apps/misc/cipher_key_value.py:39]:
```suggestion
            title = TR.misc__encrypt_value if encrypt else TR.misc__decrypt_value
            description = msg.key
            verb = TR.buttons__confirm
```

**Review by prusnak — COMMENTED** [review]
added few nit-picks:

1. let's order the assignments the same way they are called below in `confirm_action` (that is `title`, `description`, `verb`)
2. let's use `A = B if C else D` idiom for title

**Review by prusnak — APPROVED** [review]

File: core/.changelog.d/4128.added [hunk 4]
```diff
@@ -0,0 +1 @@
+Added translation of hard-coded 'Enable labeling' string
```
Comment (matejcik) [inline core/.changelog.d/4128.added:1]: this should be section "fixed" and the changelog entry is "Fix translation of the 'Enable labeling' screen."

Comment (bieleluk) [reply core/.changelog.d/4128.added]: https://github.com/trezor/trezor-firmware/pull/4128/commits/014ca79f3c7289675389a26ed2b8037d1b2e7986

**Comment by bieleluk** [comment]
Had to rebase the branch because the CI was blocked due to conflicts in the translation files

**Pull request merged**